ICAFIX/scripts/icaDim.m

```diff
@@ -31,7 +31,8 @@
 elseif DEMDT==0
     data = Origdata(OrigdataMask,:); clear Origdata;
     data_detrend = demean(data')'; meandata=mean(data')'; clear data;
-    data_trend = data_detrend.*0 + repmat(meandata,1,size(data_detrend,2));
+    data_trend = data_detrend.*0;
+    data_trend = data_trend + repmat(meandata,1,size(data_detrend,2));
```

Comment:
0 is not a large matrix, so breaking this expression isn't going to help.

Reply:
Other than the `DEMDT=1` condition, there is no point/need to form a `data_trend` time series at the front end.  At the point that `Out` is formed, we can simply have a conditional that handles the situation in a memory efficient manner at that point in the code.